blockstore/apps/api/v1/tests/test_contract.py

```diff
@@ -384,6 +384,30 @@ def test_editing_draft(self):
         )
         assert korea_text == "안녕하세요!"
 
+    def test_deleting_draft(self):
```

Comment:
This is unrelated to this PR but is temporarily included because I need it for tests. Once #40 and #49 merge, I'll rebase this PR so this unrelated diff is gone.